src/Api/Startup.cs

```diff
@@ -133,6 +135,23 @@ public void ConfigureServices(IServiceCollection services)
         services.AddDefaultServices(globalSettings);
         services.AddCoreLocalizationServices();
 
+        //health check
+        services.AddHealthCheckServices(globalSettings, builder =>
+        {
+            var identityUri = new Uri(globalSettings.BaseServiceUri.Identity
+                                      + "/.well-known/openid-configuration");
+
+            builder.AddUrlGroup(identityUri, "identity")
+                .AddRedis(globalSettings.Redis.ConnectionString)
+                .AddAzureQueueStorage(globalSettings.Storage.ConnectionString, name: "storage_queue")
+                .AddAzureQueueStorage(globalSettings.Events.ConnectionString, name: "events_queue")
+                .AddAzureQueueStorage(globalSettings.Notifications.ConnectionString, name: "notifications_queue")
+                .AddAzureServiceBusTopic(_ => globalSettings.ServiceBus.ConnectionString,
+                    _ => globalSettings.ServiceBus.ApplicationCacheTopicName, name: "service_bus")
+                .AddSendGrid(globalSettings.Mail.SendGridApiKey)
+                .AddCheck<AmazonSesHealthCheck>("amazon_ses");
```

Comment:
I think we will want checks around a lot of these so that they are only added if the various connection strings are available in the settings.